docs/Analyzer Configuration.md

```diff
@@ -792,3 +792,15 @@ Option Values: `true` or `false`
 Default Value: `false`
 
 Example: `dotnet_code_quality.CA1826.exclude_ordefault_methods = true`
+
+### Additional enum `None` names
```

Comment:
Also needs documentation in dotnet/docs (this is what I originally meant - I actually totally forgot about this file)

Reply:
https://github.com/dotnet/docs/pull/28234